### PR TITLE
Fix codeassist in sample json plugin

### PR DESCRIPTION
**Pull request by ashumilova**
Signed-off-by: Ann Shumilova ashumilova@codenvy.com

### CODE REVIEW

**Comment by ashumilova** [comment]
@azatsarynnyy, @edgarmueller please give a look

**Comment by azatsarynnyy** [comment]
LGTM

File: samples/sample-plugin-json/che-sample-plugin-json-ide/src/main/java/org/eclipse/che/plugin/jsonexample/ide/editor/JsonExampleEditorConfiguration.java [hunk 1]
```diff
@@ -31,7 +30,7 @@ public JsonExampleEditorConfiguration(@Assisted final TextEditor editor,
                                           final JsonExampleCodeAssistProcessorFactory codeAssistProcessorFactory) {
         codeAssist = new LinkedHashMap<>();
         JsonExampleCodeAssistProcessor codeAssistProcessor = codeAssistProcessorFactory.create(editor);
-        codeAssist.put(DEFAULT_CONTENT_TYPE, codeAssistProcessor);
+        codeAssist.put(DocumentPartitioner.DEFAULT_CONTENT_TYPE, codeAssistProcessor);
```
Comment (azatsarynnyy) [inline samples/sample-plugin-json/che-sample-plugin-json-ide/src/main/java/org/eclipse/che/plugin/jsonexample/ide/editor/JsonExampleEditorConfiguration.java:33]: nice catch :+1:

**Comment by benoitf** [comment]
what is target milestone for this pull request ? 4.3 ?

**Comment by slemeur** [comment]
4.3 for this.Thanks

**Pull request merged**